nox: add missing check for sync in dual view

ndellingwood opened the pull request:
Address issue #6790

 Changes to be committed:
	modified:   test/tpetra/ME_Tpetra_1DFEM_def.hpp

<!---
Be sure to select `develop` as the `base` branch against which to create this
pull request.  Only pull requests against `develop` will undergo Trilinos'
automated testing.  Pull requests against `master` will be ignored.

Provide a general summary of your changes in the Title above.  If this pull
request pertains to a particular package in Trilinos, it's worthwhile to start
the title with "PackageName:  ".

Note that anything between these delimiters is a comment that will not appear
in the pull request description once created. Most areas in this message are
commented out and can be easily added by removing the comment delimiters.

Please make sure to mark:
* Reviewers
* Assignees
* Labels

Replace <teamName> below with the appropriate Trilinos package/team name.
-->
@trilinos/nox 

## Motivation

Address issue #6790 

## Testing

Reproduced the failure in test `NOX_Tpetra_1DFEM_MPI_4` in a host build, confirmed it passes with these changes.
<!---
Please confirm that any classes or functions in the Trilinos library that this PR touches are 
exercised by at least one test in Trilinos.  Please specify which test that is.  For untestable 
changes (e.g. changes to the nightly testing system) or changes to Trilinos tests, please say "N/A".

-->

<!--- 
## Additional Information
Anything else we need to know in evaluating this merge request?
 -->

ndellingwood commented on packages/nox/test/tpetra/ME_Tpetra_1DFEM_def.hpp:
@rppawlo can you confirm if braces should wrap the code after the if-statement, similar to the `xPtr_` case below?

ndellingwood commented:
Marked WIP until confirmation regarding potential missing braces for an if-statement.

mhoemmen commented on packages/nox/test/tpetra/ME_Tpetra_1DFEM_def.hpp:
@rppawlo Why are you calling `modify_host` on the target of an Import?  It looks like this was intended to force MultiVector to pack and unpack on host.  That doesn't necessarily work like you think it does any more.  For example, the target of an Export or Import has the right to control its own sync state.

rppawlo commented on packages/nox/test/tpetra/ME_Tpetra_1DFEM_def.hpp:
@ndellingwood - the uPtr code should NOT be wrapped in braces. Each iteration we will need a new import on the solution vector (uPtr). The xPtr holds nodal coords and is not changing from iteration to iteration so only needs to be called once. We can probably remove the modify_host() calls as @mhoemmen suggested.

rppawlo commented on packages/nox/test/tpetra/ME_Tpetra_1DFEM_def.hpp:
@mhoemmen - not sure. This was written by someone who has left sandia, so I can't get a quick answer. Can we just remove the modify calls?

mhoemmen commented on packages/nox/test/tpetra/ME_Tpetra_1DFEM_def.hpp:
@rppawlo We should remove them, unless downstream code implicitly depends on MultiVector being sync'd to host.  There's no good way to test for that, though.

ndellingwood commented on packages/nox/test/tpetra/ME_Tpetra_1DFEM_def.hpp:
> We should remove them, unless downstream code implicitly depends on MultiVector being sync'd to host

@rppawlo @mhoemmen I'll hold off making any changes until getting a definite "yes" to remove the `modify_host` calls (and the sync checking I added in this PR)

ndellingwood commented on packages/nox/test/tpetra/ME_Tpetra_1DFEM_def.hpp:
@rppawlo thanks for clarifcation.

rppawlo commented on packages/nox/test/tpetra/ME_Tpetra_1DFEM_def.hpp:
yes - let's remove them.

ndellingwood commented:
@rppawlo @mhoemmen I removed the sync changes I introduced, removed the unnecessary `modify_host` calls, and changed indentation of the uPtr_ import call to prevent potential warnings about misleading indentation. Rebased the changes for a clean single commit

trilinos-autotester commented:
Status Flag 'Pre-Test Inspection' - Auto Inspected - Inspection Is Not Necessary for this Pull Request.

trilinos-autotester commented:
Status Flag 'Pull Request AutoTester' - Testing Jenkins Projects:

<details>
<summary>
  <b>Pull Request Auto Testing STARTING</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_gcc_4.8.4</h3>
<ul>
  <li>Build Num: 5693</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>COMPILER_MODULE</td>
    <td>sems-gcc/4.8.4</td>
  </tr>

  <tr>
    <td>JENKINS_BUILD_TYPE</td>
    <td>Release</td>
  </tr>

  <tr>
    <td>JENKINS_COMM_TYPE</td>
    <td>MPI</td>
  </tr>

  <tr>
    <td>JENKINS_DO_COMPLEX</td>
    <td>OFF</td>
  </tr>

  <tr>
    <td>JENKINS_JOB_TYPE</td>
    <td>Experimental</td>
  </tr>

  <tr>
    <td>MPI_MODULE</td>
    <td>sems-openmpi/1.8.7</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>6793</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>issue-6790</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>221a9c839d553143b386b15c78c261c85977f3d8</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>68b1136da32ca24f1e1ed2d203529fcccc122912</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_intel_17.0.1</h3>
<ul>
  <li>Build Num: 5518</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>6793</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>issue-6790</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>221a9c839d553143b386b15c78c261c85977f3d8</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>68b1136da32ca24f1e1ed2d203529fcccc122912</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_gcc_4.9.3_SERIAL</h3>
<ul>
  <li>Build Num: 3947</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>6793</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>issue-6790</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>221a9c839d553143b386b15c78c261c85977f3d8</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>68b1136da32ca24f1e1ed2d203529fcccc122912</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_gcc_7.2.0</h3>
<ul>
  <li>Build Num: 3795</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>6793</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>issue-6790</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>221a9c839d553143b386b15c78c261c85977f3d8</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>68b1136da32ca24f1e1ed2d203529fcccc122912</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_cuda_9.2</h3>
<ul>
  <li>Build Num: 3332</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>JENKINS_JOB_TYPE</td>
    <td>Experimental</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>6793</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>issue-6790</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>221a9c839d553143b386b15c78c261c85977f3d8</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>68b1136da32ca24f1e1ed2d203529fcccc122912</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_python_2</h3>
<ul>
  <li>Build Num: 1600</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>6793</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>issue-6790</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>221a9c839d553143b386b15c78c261c85977f3d8</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>68b1136da32ca24f1e1ed2d203529fcccc122912</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_python_3</h3>
<ul>
  <li>Build Num: 1599</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>6793</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>issue-6790</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>221a9c839d553143b386b15c78c261c85977f3d8</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>68b1136da32ca24f1e1ed2d203529fcccc122912</td>
  </tr>
</table>

<h2>Using Repos:</h2>

<dl>
  <dt>Repo: TRILINOS (trilinos/Trilinos)</dt>
  <ul>
    <li>Branch: issue-6790</li>
    <li>SHA: 221a9c839d553143b386b15c78c261c85977f3d8</li>
    <li>Mode: TEST_REPO</li>
  </ul>
</dl>

Pull Request Author: ndellingwood
</details>

mhoemmen commented:
Thanks @ndellingwood ! :-D

trilinos-autotester commented:
Status Flag 'Pull Request AutoTester' - Jenkins Testing: all Jobs PASSED

<details>
<summary>
  <b>Pull Request Auto Testing has PASSED</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_gcc_4.8.4</h3>
<ul>
  <li>Build Num: 5693</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>COMPILER_MODULE</td>
    <td>sems-gcc/4.8.4</td>
  </tr>

  <tr>
    <td>JENKINS_BUILD_TYPE</td>
    <td>Release</td>
  </tr>

  <tr>
    <td>JENKINS_COMM_TYPE</td>
    <td>MPI</td>
  </tr>

  <tr>
    <td>JENKINS_DO_COMPLEX</td>
    <td>OFF</td>
  </tr>

  <tr>
    <td>JENKINS_JOB_TYPE</td>
    <td>Experimental</td>
  </tr>

  <tr>
    <td>MPI_MODULE</td>
    <td>sems-openmpi/1.8.7</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>6793</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>issue-6790</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>221a9c839d553143b386b15c78c261c85977f3d8</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>68b1136da32ca24f1e1ed2d203529fcccc122912</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_intel_17.0.1</h3>
<ul>
  <li>Build Num: 5518</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>6793</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>issue-6790</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>221a9c839d553143b386b15c78c261c85977f3d8</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>68b1136da32ca24f1e1ed2d203529fcccc122912</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_gcc_4.9.3_SERIAL</h3>
<ul>
  <li>Build Num: 3947</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>6793</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>issue-6790</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>221a9c839d553143b386b15c78c261c85977f3d8</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>68b1136da32ca24f1e1ed2d203529fcccc122912</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_gcc_7.2.0</h3>
<ul>
  <li>Build Num: 3795</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>6793</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>issue-6790</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>221a9c839d553143b386b15c78c261c85977f3d8</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>68b1136da32ca24f1e1ed2d203529fcccc122912</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_cuda_9.2</h3>
<ul>
  <li>Build Num: 3332</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>JENKINS_JOB_TYPE</td>
    <td>Experimental</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>6793</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>issue-6790</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>221a9c839d553143b386b15c78c261c85977f3d8</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>68b1136da32ca24f1e1ed2d203529fcccc122912</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_python_2</h3>
<ul>
  <li>Build Num: 1600</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>6793</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>issue-6790</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>221a9c839d553143b386b15c78c261c85977f3d8</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>68b1136da32ca24f1e1ed2d203529fcccc122912</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_python_3</h3>
<ul>
  <li>Build Num: 1599</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>6793</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>issue-6790</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>221a9c839d553143b386b15c78c261c85977f3d8</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>68b1136da32ca24f1e1ed2d203529fcccc122912</td>
  </tr>
</table>
</details>


<br>[CDash Test Results for PR# 6793.](https://testing.sandia.gov/cdash/index.php?project=Trilinos&filtercombine=and&filtercombine=and&filtercombine=and&filtercombine=and&filtercombine=and&filtercombine=and&filtercount=2&showfilters=1&filtercombine=and&field1=buildname&compare1=63&value1=PR-6793-test&field2=buildstarttime&compare2=84&value2=NOW
)<br>

trilinos-autotester commented:
Status Flag 'Pre-Merge Inspection' - SUCCESS: The last commit to this Pull Request has been INSPECTED AND APPROVED by [ mhoemmen rppawlo ]!

trilinos-autotester commented:
Status Flag 'Pull Request AutoTester' - Pull Request will be Automerged

trilinos-autotester commented:
Merge on Pull Request# 6793: IS A SUCCESS - Pull Request successfully merged
